### ISSUE TITLE
Domain user auth Failing

### ISSUE THREAD

**Issue by clumsypuffin** [microsoft/mssql-django#35]
Hi

I'm able to use Sql user to communicate with the DB but using domain account is causing error:

"[28000] [Microsoft][ODBC Driver 11 for SQL Server][SQL Server]Login failed for user 'domain\\user'. (18456) Invalid connection string attribute (0)"

These are the configuration that I'm using:

`DATABASES = {
    'default': {
        'ENGINE': 'mssql',
        'NAME': 'mydb',
        'USER': 'domain\\user',   # I've checked with domain\user as well user@domain
        'PASSWORD': 'pass',
        'HOST': 'myserver',
        'PORT': '',
        'OPTIONS': {
            'driver': 'ODBC Driver 11 for SQL Server',
        },

    },
}`

Am I missing something obvious ,Can you help me out with this ?

**Comment by absci** [comment]
Have you tried [`sqlcmd`](https://docs.microsoft.com/en-us/sql/ssms/scripting/sqlcmd-connect-to-the-database-engine?view=sql-server-ver15) command to that domain?
`ODBC Driver 11 for SQL Server` seems very old, is it possible to update it to [ODBC 17](https://docs.microsoft.com/en-us/sql/connect/odbc/download-odbc-driver-for-sql-server?view=sql-server-ver15)?

**Comment by clumsypuffin** [comment]
@absci yes,I usually use domain login with the integrated option for sqlcmd.  I understand that it's not officially supported (the SQL server version), Will check about the upgrade.

**Comment by clumsypuffin** [comment]
@absci I have updated to ODBC 17 (django 3.2 and mssql-django v1) and the domain user issue is still there. any suggestions ?

**Comment by absci** [comment]
I'll look into this issue, could you share the command that you used with `sqlcmd` when you successfully logged in?

**Comment by absci** [comment]
Any updates or additional information? We'll close this issue next week if we don't hear back. Feel free to reopen.
